certified-connectors/InEight/apiDefinition.swagger.json

```diff
@@ -3,7 +3,7 @@
   "info": {
     "version": "1.0",
     "title": "InEight",
-    "description": "The InEight Connector provides a mechanism to create new data in InEight or extract data from InEight. It can be used to integrate other systems with InEight.",
+    "description": "InEight Connector provides an API to work with InEight objects.",
```

Comment:
@InEightDevOps Please revert the description value back as the existing one looks good than the new one.

Reply:
Updated comments as requested.